shared/reducers/settings.js

```diff
@@ -257,6 +263,7 @@ function reducer(state: Types.State = Constants.initialState, action: SettingsGe
     case SettingsGen.processorProfile:
     case SettingsGen.unfurlSettingsRefresh:
     case SettingsGen.loadHasRandomPw:
+    case SettingsGen.testPassphrase:
```

Comment:
this should actually set testPasspraseIsCorrect to null to indicate we're in the act of determining it